PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift

```diff
@@ -25,8 +25,6 @@ import Foundation
     @objc public var paymentqueue_revoked_entitlements_for_product_identifiers: String { "PaymentQueue " +
         "didRevokeEntitlementsForProductIdentifiers: %@" }
     @objc public var paymentqueue_updatedtransaction: String { "PaymentQueue updatedTransaction: %@ %@ (%@) %@ - %d" }
-    @objc public var presenting_code_redemption_sheet_unavailable: String { "Attempted to present code redemption " +
-        "sheet, but it's not available on this device." }
```

Comment:
this wasn't really necessary - we were using it on a method that was marking availability anyway